feast-dev/feast
# chore: Removed unneeded install line

# What this PR does / why we need it:

This PR removes the command `pip install Pygments -q` (and a supporting comment) from the `quickstart.ipynb` notebook. This install command is not needed because `Pygments` is a `feast` dependency that is installed with `feast`.  Removing the extra install line simplifies the notebook and makes it easier to get started quickly, as it makes the cell run faster.

# Which issue(s) this PR fixes:



# Misc

To test that the notebook still works you can:
1. Create a new venv and install jupyter (you need this to run the notebook):
```#bash
cd examples/quickstart
python -m venv temp-venv
pip install jupyter
```

2. Run the `quickstart.ipynb` from this PR (with the Pygments import statement removed). It should run successfully.
```#bash
jupyter execute quickstart.ipynb
```
Note that the notebook itself installs `feast` latest. You can verify that `pygments` is an explicit dependency of feast with:
```#bash
% pip show Pygments | grep Required-by

# Output
Required-by: feast, ipython, jupyter-console, nbconvert
```
